tests/Properties.hs

```diff
@@ -93,6 +96,23 @@ prop_lines_lazy2 =
 
 prop_strip x = C.strip x == (C.dropSpace . C.reverse . C.dropSpace . C.reverse) x
 
+prop_toConstr :: P.ByteString -> Property
+prop_toConstr bs = True ==> "pack" == ((showConstr  . toConstr) bs)
+
+prop_gshow_empty :: P.ByteString -> Property
+prop_gshow_empty b = (not . null . Char8.unpack) b ==> (not . null . gshow) b
```

Comment:
I don't understand this property. Just like the output for `show ` is never empty, `gshow` output shouldn't _ever_ be empty, too, right? Therefore the precondition can be dropped, but I also don't really understand the purpose of the test.

Reply:
It was just there to show that for any byte string as input the gshow function should not produce an error, but after adding more tests to this that cover the same idea and test other things I don't think it is needed anymore.